release 9.0.13

welansari opened the pull request:
Issue: BB-700

Commit:
release 9.0.13

Issue: BB-700

bert-e commented:
# Hello kerkesni,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/backbeat/doc/user).


<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Request integration branches



Waiting for integration branch creation to be requested by the user.

To request integration branches, please comment on this pull request with the following command:

```
/create_integration_branches
```

Alternatively, the `/approve` and `/create_pull_requests` commands will automatically
create the integration branches.

welansari commented:
/approve

bert-e commented:
# Conflict




A conflict has been raised during the creation of
integration branch `w/9.1/improvement/BB-700` with contents from `improvement/BB-700`
and `development/9.1`.

**I have not created the integration branch.**


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout -B w/9.1/improvement/BB-700 origin/development/9.1
 $ git merge origin/improvement/BB-700
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/9.1/improvement/BB-700
```





*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers











*The following options are set:* **approve**

bert-e commented:
# Build failed



The [build](https://github.com/scality/backbeat/actions?query=branch%3Aimprovement/BB-700) for [commit](https://github.com/scality/backbeat/commit/23b10bf6dc7d7e6f3a252e13869a4b62d9467131) did not succeed in branch [improvement/BB-700](https://github.com/scality/backbeat/compare/development/9.0...improvement/BB-700)



*The following options are set:* **approve**

Commit:
fix ft tests

Debian Buster repositories are no longer available.
Debian Buster reached end-of-life and has been moved to archive repositories

Issue: BB-697

bert-e commented:
# History mismatch



Merge commit #23b10bf6dc7d7e6f3a252e13869a4b62d9467131 on the integration branch
`w/9.1/improvement/BB-700` is merging a branch which is neither the current
branch `improvement/BB-700` nor the development branch
`development/9.1`.

It is likely due to a rebase of the branch `improvement/BB-700` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

welansari commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve**

bert-e commented:
# Build failed



The [build](https://github.com/scality/backbeat/actions?query=branch%3Aimprovement/BB-700) for [commit](https://github.com/scality/backbeat/commit/2abfc06cb134b9c8a85d10fd3f6e43b312980987) did not succeed in branch [improvement/BB-700](https://github.com/scality/backbeat/compare/development/9.0...improvement/BB-700)



*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/9.0`

* :heavy_check_mark: `development/9.1`



The following branches have **NOT** changed:

* `development/7.10`
* `development/7.4`
* `development/7.70`
* `development/8.6`



Please check the status of the associated issue BB-700.

Goodbye kerkesni.




*The following options are set:* **approve**
